pkgs/by-name/ku/kubectl-kcl/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "kubectl-kcl";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "kcl-lang";
+    repo = "kubectl-kcl";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-yuNQSO1xQCb5H55mOUTVrojeWWkDOmAGJIzUs6qCWO4=";
+  };
+
+  vendorHash = "sha256-GD4C4jlxVMpJ/bhpQ3VDkBMBBQkXyhMMga+WhVdvI/I=";
+
+  ldflags = [
+    "-X kcl-lang.io/kubectl-kcl/cmd.Version=${finalAttrs.version}"
+  ];
+
+  versionCheckProgramArg = [ "version" ];
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  doInstallCheck = true;
+
+  meta = {
+    description = "Kubectl KCL Plugin";
```

```suggestion
    description = "Work with Kubernetes manifests using the KCL programming language";
```
description should not just be the package name.